### ISSUE TITLE
How to use with woocommerce please

### ISSUE THREAD

**Issue by SignedT** [vijayhardaha/wp-ajaxify#1]
Hi your doc files for woo seem to be missing: [https://github.com/vijayhardaha/wp-ajaxify/blob/master/docs/woocommerce.md](url)
Thanks

**Comment by SignedT** [comment]
Also will this be updated in Wordpress admin when updates are available?

**Comment by vijayhardaha** [comment]
Yes, WooCommerce documentation is not ready yet. Maybe in the future, I will write down the required steps for proper WooCommerce configuration, However, you can read Ajaxify.js Documentation and manage the settings as per your needs.

And Automatic updates won't show in your WordPress Admin area, this WordPress plugin is only available on GitHub repository.
